test/rubygems/test_gem_request_set.rb

```diff
@@ -396,9 +396,9 @@ def test_resolve_git
     rs = Gem::RequestSet.new
 
     tf = Tempfile.open 'gem.deps.rb' do |io|
-      io.puts <<-gems_deps_rb
+      io.puts <<-GEMS_DEPS_RB
```

Comment:
If I understand it well, this is also just code ruby block. Any reason to not use `RUBY` as well?